src/js/yeldaWebchatInjector.js

```diff
@@ -775,40 +775,109 @@ class YeldaChat {
     return isFound
   }
 
+  /**
+   * Get the app env from the window url
+   * @param {String} href 
+   * @returns {String} env
+   */
+  getAppEnv(href) {
+    if (config.REGEX_HOST.STAGING.test(href)) {
+      return 'staging'
+    }
+    if (config.REGEX_HOST.LOCALHOST.test(href)) {
+      return 'local'
+    }
+    return 'prod'
+  }
+
+  /**
+   * Get the chat bubble url base based on isFallback
+   * @param {Object} data { assistantUrl, assistantId, settingId }
+   * @param {Boolean} isFallback fallback means yelda endpoint
+   * @returns {String} url base
+   */
+  getChatBubbleUrlBase(data, isFallback) {
+    if (isFallback) {
+      return `${data.assistantUrl}/assistants`
+    }
+    return 'https://webchat.yelda.ai/webchat'
+  }
+
   /**
    * return getAssistantSettings endpoint URL from data
    * @param {Object} data { assistantUrl, assistantId, settingId }
+   * @param {String} urlBase
    * @return {String} url
    */
-  getAssistantSettingsUrl(data) {
+  getAssistantSettingsUrl(data, urlBase) {
+    // The app env is required as query for the external chatbubble endpoint
+    const appEnv = this.getAppEnv(window.location.href)
```

Comment:
:gear: I would move the `getChatBubbleUrlBase` into `getAssistantSettingsUrl` because this is the only place where is used. No need to have `urlBase` in `getAssistantSettings` context